Tensile/KernelWriterAssembly.py

```diff
@@ -3420,6 +3420,12 @@ def graTileAssignment(self, kernel, tP):
       dummy       = self.vgprPool.checkOut(1, "dummy", self.preventVgprOverflowDuringNewTile)
       kStr += vectorStaticRemainder(dummy, dividendReg, "Serial", self.kernel["WavefrontSize"], tmpVgpr, tmpSgpr)
 
+    splitRead = kernel["SplitGlobalRead"]
+    # Split global read reorders reading rows within lanes of a wavefront
+    # If the wavefront is reading all from a single row, then disable split global read for this tensor
+    if divisor > self.kernel["WavefrontSize"]:
```

Comment:
We do not actually need "self." here....
Kernel is passed as a parameter.

Reply:
Fixed